runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/DataflowRunnerInfoTest.java

```diff
@@ -49,8 +49,19 @@ public void getDataflowRunnerInfo() throws Exception {
         String.format("FnAPI environment major version number %s is not a number", version),
         version.matches("\\d+"));
 
-    // Validate container version does not contain a $ (indicating it was not filled in).
-    assertThat("container version invalid", info.getContainerVersion(), not(containsString("$")));
+    // Validate container version does not contain the property name (indicating it was not filled
+    // in).
+    assertThat(
```

Comment:
Previous default property (in dataflow.properties) was `$(...)` so current test won't test anything; using the full property string s.t. hopefully someone will find it if they decide to change the default property value.